lib/rules/selector-not-notation/index.mjs

```diff
@@ -79,46 +81,109 @@ const rule = (primary, _, context) => {
 
 			if (!selectorRoot) return;
 
+			/** @param {Pseudo} not */
+			const fixSimple = (not) => {
+				const simpleSelectors = not.nodes
+					.filter(({ nodes }) => nodes[0] && isSimpleSelector(nodes[0]))
+					.map((s) => {
+						assert(s.nodes[0]);
+						s.nodes[0].rawSpaceBefore = '';
+						s.nodes[0].rawSpaceAfter = '';
+
+						return s;
+					});
+				const firstSelector = simpleSelectors.shift();
+				let offset = 0;
```

Comment:
note

This is the only notable change that I have done to the existing fixers.